WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -482,39 +482,36 @@ function(task, colIdx) {
 	var width = (AjxEnv.isIE || AjxEnv.isSafari) ? "22" : "16";
 
 	// first row
-	htmlArr[idx++] = "<table width=100% class='TopRow'>";
+	htmlArr[idx++] = "<table class='TopRow'>";
 	htmlArr[idx++] = "<tr id='";
 	htmlArr[idx++] = DwtId.getListViewItemId(DwtId.WIDGET_ITEM_FIELD, this._view, task.id, ZmItem.F_ITEM_ROW_3PANE);
 	htmlArr[idx++] = "'>";
 
-    idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_SUBJECT, colIdx);
+	if (task.pComplete) {
+		htmlArr[idx++] = "<td class='ZmTaskHeaderImage'>";
+		htmlArr[idx++] = AjxImg.getCircularProgressBar(task.pComplete);
+		htmlArr[idx++] = "</td>";
+	}
 
-    idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_DATE, colIdx, ZmMsg.COLUMN_WIDTH_DATE);
+	idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_SUBJECT, colIdx);
 
-	htmlArr[idx++] = "</tr></table>";
+	htmlArr[idx++] = "<td align='right'><table><tr>";
+	idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_DATE, colIdx, ZmMsg.COLUMN_WIDTH_DATE);
+	htmlArr[idx++] = "</tr><tr><td><table class='BottomRow'><tr>";
 
-    // second row
-    htmlArr[idx++] = "<table width=100% class='BottomRow'><tr><td>";
-	if (task.pComplete) {
-		htmlArr[idx++] = "<div class='ZmTaskProgress'><div";
-		htmlArr[idx++] = " class='";
-		htmlArr[idx++] = this.getColorForStatus(task.status);
-		htmlArr[idx++] = "' style='width:"+ task.pComplete + "%;'></div></div>";
+	idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_TAG, colIdx, width, "align=right");
+	if (task.priority == ZmCalItem.PRIORITY_HIGH || task.priority == ZmCalItem.PRIORITY_LOW) {
+		idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_PRIORITY, colIdx, width, "align=right");
 	}
-    htmlArr[idx++] = "</td><td><table class='IconInnerWrapper'><tr>";
 
-    idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_TAG, colIdx, width);
-    if(task.priority == ZmCalItem.PRIORITY_HIGH || task.priority == ZmCalItem.PRIORITY_LOW) {
-        idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_PRIORITY, colIdx, width, "align=right");
-    }
-    if (task.hasAttach) {
-        idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_ATTACHMENT, colIdx, width);
-    }
-    htmlArr[idx++] = "</tr></table></td>";
-    htmlArr[idx++] = "</tr></table>";
+	if (task.hasAttach) {
+		idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_ATTACHMENT, colIdx, width);
+	}
 
+	htmlArr[idx++] = "</tr></table></td>";
+	htmlArr[idx++] = "</tr></table></td>";
```

Comment:
Not really, I mean that  there were two closings for table tag before and now I see 3 closings. I don't see any new addition of new table tag opening. Same with <tr> , I believe there are 2 rows (topRow & bottomRow) and I see  3 closing for that.
Correct me if I misunderstood something here.